ContributionGuidelines.md

```diff
@@ -0,0 +1,140 @@
+# [DOC2PEN](https://doc2pen.smaranjitghose.codes/)📝
+#### Doc2pen is an Open Source Software that converts your Typed Docs into Handwritten Docs.
+
+# Contribution Guidelines [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/dwyl/esta/issues) 🤝🏽🍀:
+
+## File Structure of the Project 🗃️📂
+
+### [Public Directory Structure](https://github.com/smaranjitghose/doc2pen/tree/master/public) 🧮✨
+This folder contains the stylesheets, images and fonts. Any files within this directory will not be processed by Webpack but copied directly to the build folder.
+
+Files/Folders that are directly located inside the Public Folder: 📥
+- favicon.ico
+- images
+- index.html
+- logo.png
+- logo192.png
+- logo512.png
+- manifest.json
+- _redirects
+- robots.txt
+
+
+### [Src Directory Structure](https://github.com/smaranjitghose/doc2pen/tree/master/src) 🧮✨
+This folder contains all the main source code for the React application.
+
+Files/Folders that are directly located inside the Src Folder: 📥
+- app.js 
+- app.test.js
```

Comment:
It is better to explain this with a tree structure:

|-src
     |-... 
     |- 
        _- 

Now explain each one

Reply:
Sure